testsuite/integration-arquillian/servers/auth-server/quarkus/src/main/content/conf/keycloak.properties

```diff
@@ -1,24 +1,20 @@
 hostname.default.frontendUrl = ${keycloak.frontendUrl:}
 
-# Datasource
-datasource.jdbc.transactions=xa
-
 # H2
-hibernate-orm.dialect=org.hibernate.dialect.H2Dialect
-datasource.driver=org.h2.jdbcx.JdbcDataSource
-datasource.url = jdbc:h2:file:${keycloak.home.dir}/data/keycloakdb;AUTO_SERVER=TRUE;DB_CLOSE_DELAY=-1
-datasource.username = sa
-datasource.password = keycloak
+database.vendor=h2
+database.url = jdbc:h2:file:${keycloak.home.dir}/data/keycloakdb;AUTO_SERVER=TRUE;DB_CLOSE_DELAY=-1
+database.username = sa
+database.password = keycloak
 
 # SSL
-http.ssl.certificate.key-store-file=${keycloak.home.dir}/conf/keycloak.jks
-http.ssl.certificate.key-store-password=secret
-http.ssl.certificate.trust-store-file=${keycloak.home.dir}/conf/keycloak.truststore
-http.ssl.certificate.trust-store-password=secret
-http.ssl.client-auth=REQUEST
+https.certificate.key-store-file=${keycloak.home.dir}/conf/keycloak.jks
+https.certificate.key-store-password=secret
+https.certificate.trust-store-file=${keycloak.home.dir}/conf/keycloak.truststore
+https.certificate.trust-store-password=secret
+https.client-auth=REQUEST
 
 # Proxy
-http.proxy-address-forwarding=true
+quarkus.http.proxy-address-forwarding=true
```

Comment:
Should this be changed to "proxy.mode" as mentioned in https://github.com/keycloak/keycloak-community/blob/master/design/keycloak.x/configuration.md#proxy-mode .

Reply:
Yes, but as stated in the description this is not implemented yet.

Reply:
Ok